config/settings.yml

```diff
@@ -105,6 +105,17 @@ homepage_navigation:
       path: /cases/closed
       scope: closed_cases
 
+    case_retention:
+      path: /cases/retention
+      scope: retention_cases
+      visibility:
+        - BRANSTON
```

Comment:
The visibility will be restricted to certain group further, not sure what Nicole decided, either `manager` ( but there is manager so far for Branston team) or `team_admin` (maybe this one is more suitable).  But this one can be defined in the future PR. ---> my comment here is just for note